lib/Models/esriLineStyle.ts

```diff
@@ -0,0 +1,43 @@
+const defaultDashArray = [4, 3];
+
+export const esriLineStyleCesium: {
```

Comment:
```suggestion
const esriLineStyleCesium: {
```

forgot to remove export in the last commit (this is only used in this file)